inlong-sdk/sort-sdk/src/test/java/org/apache/inlong/sdk/sort/impl/decode/MessageDeserializerTest.java

```diff
@@ -165,6 +169,33 @@ private void testDeserializeVersion1CompressionType2() {
         }
     }
 
+    private void testDeserializeVersion2NoCompress() {
+        try {
+            String groupId = "sort_sdk_test_group_id";
+            String streamId = "sort_sdk_test_stream_id";
+            String attr = "m=0";
+            String ip = "1.2.3.4";
+            long dt = System.currentTimeMillis();
+            StringBuilder newAttrBuffer = new StringBuilder(attr);
```

Comment:
Why name it `buffer`?

Reply:
fixed, rename it to **_newAttrBuilder_**